reframe/core/pipeline.py

```diff
@@ -1328,9 +1328,9 @@ def run(self):
         if not self.current_system or not self._current_partition:
             raise PipelineError('no system or system partition is set')
 
+        prepare_run = []
```

Comment:
I want to combine this with with the `prepare_container` commands. @teojgo Is there a reason that the `prepare_container` gets appended to `prerun_cmds`. Could it be prepended instead?